test/generic/MPoly-test.jl

```diff
@@ -140,6 +140,17 @@
    QQxxx3 = @polynomial_ring(QQ, :x=>1:3)
    @test QQxxx_ == (QQxxx3, [x1, x2, x3])
 
+   # test support of kwargs
+   QQxxx4 = @polynomial_ring(QQ, "x#" => 1:3; internal_ordering=:lex)
```

Comment:
Why did you change the symbol from `:x` to `x#`?

Reply:
So that I can directly compare the result of `polynomial_ring` and `@polynomial_ring`: the former turns `:x => 1:3` into variable `x[1]` etc., the latter into `x1`, etc.